### PR TITLE
docs: update README with failover info

### PR DESCRIPTION
**Pull request by samlaf**
## Fixes Issue

Fixes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,12 +107,10 @@ An optional storage fallback CLI flag `--routing.fallback-targets` can be levera
 ### Storage Caching
 An optional storage caching CLI flag `--routing.cache-targets` can be leveraged to ensure less redundancy and more optimal reading. When enabled, a blob is persisted to each cache target after being successfully dispersed using the keccak256 hash of the existing EigenDA commitment for the fallback target key. This ensure second order keys are succinct. Upon a blob retrieval request, the cached targets are first referenced to read the blob data before referring to EigenDA. 
 
+### Failover Signals
+In the event that the EigenDA disperser or network is down, the proxy will return a 503 status code as a response to POST requests, which rollup batchers can use to failover and start submitting blobs to the L1 chain instead. See https://github.com/ethereum-optimism/specs/issues/434 for more info in the context of the [op stack](#optimism-commitment-mode).
```
Comment (epociask) [inline README.md:111]: Let's also add the arb spec: https://hackmd.io/@epociask/SJUyIZlZkx

File: README.md [hunk 2]
```diff
@@ -107,12 +107,10 @@ An optional storage fallback CLI flag `--routing.fallback-targets` can be levera
...
 
-## Metrics
-
-To the see list of available metrics, run `./bin/eigenda-proxy doc metrics`
-
-To quickly set up monitoring dashboard, add eigenda-proxy metrics endpoint to a reachable prometheus server config as a scrape target, add prometheus datasource to Grafana to, and import the existing [Grafana dashboard JSON file](./grafana_dashboard.json)
+This behavior is turned on by default, but configurable via the `--eigenda.confirmation-timeout` flag (set to 15 mins by default currently). If a blob is not confirmed within this time, the proxy will return a 503 status code. This should be set long enough to accomodate for the disperser's batching interval (typically 10 minutes), signature gathering, and onchain submission.
```
Comment (epociask) [inline README.md:113]: knit - should we also specify the status code meaning? `ServiceUnavailable`

File: README.md (hunk 1 above)
Comment (samlaf) [reply README.md]: [9e722cf](https://github.com/Layr-Labs/eigenda-proxy/pull/204/commits/9e722cff2ad07b9a0246dc2e34eb8ebb41823cc0)

File: README.md (hunk 2 above)
Comment (samlaf) [reply README.md:113]: [6244e72](https://github.com/Layr-Labs/eigenda-proxy/pull/204/commits/6244e72aa9c134e21000d46d5880369a468b6551)

**Review by epociask — APPROVED** [review]
LGTM

**Pull request merged**